modules/apps/object/object-test/src/testFunctional/macros/ObjectAdmin.macro

```diff
@@ -1394,6 +1394,12 @@ definition {
 			locator1 = "ObjectAdmin#VIEW_PICKLIST_OPTIONS");
 	}
 
+	macro viewSectionTitle {
```

Comment:
This is not needed. One step macros can be on the testcase level especially if it's an assertion. Avoid having assertions in macros.